state/l1infotree.go

```diff
@@ -0,0 +1,64 @@
+package state
+
+import (
+	"context"
+
+	"github.com/0xPolygonHermez/zkevm-node/l1infotree"
+	"github.com/0xPolygonHermez/zkevm-node/log"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jackc/pgx/v4"
+)
+
+// L1InfoTreeLeaf leaf of the L1InfoTree
+type L1InfoTreeLeaf struct {
+	GlobalExitRoot
+	PreviousBlockHash common.Hash
+}
+
+// L1InfoTreeExitRootStorageEntry entry of the Database
+type L1InfoTreeExitRootStorageEntry struct {
+	L1InfoTreeLeaf
+	L1InfoTreeRoot  common.Hash
+	L1InfoTreeIndex uint64
+}
```

Comment:
Why do we need a struct with three levels?? I think this is unnecessary. I think two levels are more than enough

Reply:
@ARR552  Do you mean don't reuse the `GlobalExitRoot` struct o merge the `L1InfoTreeLeaf` and `L1InfoTreeExitRootStorageEntry`?